site/content/pages/organizing/01-Getting started/04-requirements.txt

```diff
@@ -0,0 +1,18 @@
+---
+extension: html
+filter:
+- erb
+- markdown
+title: Rules
+---
+
+Every devopsdays event is different, but there are a few rules to keep in mind if you'd like your event to be listed on devopsdays.org:
+
+- Inclusiveness and respect for differences are core devops values, and we invite you to help us make each devopsdays event a place that is welcoming and respectful to all participants. Your event will need to have a code of conduct. 
+- These are community events, so your event must accept registrations from the general public. Internal devops events focused on a specific organization are wonderful and we encourage them, but they won't be listed as "devopsdays" events on this site.
+- These events are not for individual or corporate profit. If you have money left over, you can use it for your next event, to help other devopsdays events, or for charity. The global core team does not have any ability to accept your money, but neither should it be making its way into anyone's pockets. :)
```

Comment:
@bridgetkromhout I wonder if we should have a stipulation about sharing the financial info back with the core team, we have previously collected some money for things like designers, web stuff etc.. It would be interesting to investigate the mechanism to do that again, however probably a discussion for the list, outside the scope of this PR..

I'd probably drop the smiley face here also, in case it is misuderstood as the rule not being serious (I realize that sounds dumb.. just being paranoid)